### PR TITLE
Multi-language support (zh/en/ja) with nuxt-i18n

### PR DESCRIPTION
**Pull request by itTkm**
@lianginx Thank you for your great project.

I have implemented multi-language support by introducing nuxt-i18n.
Please check it out.

### CODE REVIEW

**Comment by lianginx** [comment]
Wow! Thank you very much for your code. I will merge it after the test!

Thank you again for your contribution. You have helped me achieve the small goal of "changing the world"!

File: locales/en.json [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+      "examples": [
+        {
+          "title": "How to send HTTP requests using Javascript?",
+          "message": {
+            "role": "user",
+            "content": "How to send HTTP requests using Javascript?"
+          }
+        },
+        {
+          "title": "Translation: 翻译你好，快乐的世界！",
```
Comment (lianginx) [inline locales/en.json:51]:
```suggestion
          "title": "Translation: 你好，快乐的世界！",
```

File: locales/en.json [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+          "message": {
+            "role": "user",
+            "content": "Translation: 翻译你好，快乐的世界！"
```
Comment (lianginx) [inline locales/en.json:54]:
```suggestion
            "content": "Translation: 你好，快乐的世界！"
```

**Review by lianginx — CHANGES_REQUESTED** [review]
Thank you for your contribution, it's great! Just two translations need to be modified.

File: locales/en.json (hunk 1 above)
Comment (itTkm) [reply locales/en.json]: fixed: 803e0fd

File: locales/en.json (hunk 2 above)
Comment (itTkm) [reply locales/en.json]: fixed: 803e0fd

**Review by lianginx — APPROVED** [review]

**Comment by lianginx** [comment]
I found an issue, after running `yarn build`, `node .output/server/index.mjs` will throw an error.

image

**Pull request merged**